openmdao/components/exec_comp.py

```diff
@@ -441,6 +462,8 @@ def _setup_expressions(self):
                 if var in outs:
                     current_meta = self.add_output(var, val, **meta)
                 else:
+                    if 'constant' in meta:
+                        meta.pop('constant', None)
```

Comment:
I assume that if 'constant' is in meta here that it has to be False, right?

Reply:
yes